wingbits/Dockerfile.template

```diff
@@ -14,7 +14,7 @@ ENV WINGBITS_CONFIG_VERSION=0.0.8
 ENV WINGBITS_COMMIT_ID=f8091ec5
 ENV WINGBITS_DATE=2025-01-17T12:07:51.885884718Z
 
-ARG PERM_INSTALL="curl gettext-base tini ncurses-dev zlib1g jq" 
+ARG PERM_INSTALL="curl gettext-base tini ncurses-dev zlib1g jq python3 python3-pip python3-venv wget"
```

Comment:
```suggestion
ARG PERM_INSTALL="curl gettext-base tini ncurses-dev zlib1g jq python3-pip python3-venv wget"
```

Reply:
Correct. I'll test it now just to be sure 👌

Reply:
balena not liking me today...getting errors when trying to build

Reply:
working great - merging